__tests__/app.test.js

```diff
@@ -0,0 +1,81 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+
+beforeEach(() => {
+	return seed(testData);
+});
+
+afterAll(() => {
+	return db.end();
+});
+
+describe('Topics', () => {
+    
+    test('GET /api/topics should respond with a 200 status code', () => {
```

Comment:
This status test is implicit in any other tests that assert the status code so doesn't need to be a stand alone test - you can just include it as you are doing in the othes